dipy/reconst/dki.py

```diff
@@ -1259,7 +1516,7 @@ def dki_prediction(dki_params, gtab, S0=1.):
         else:
             this_S0 = S0_vol
         X = np.concatenate((dt, fkt[v] * MD * MD,
-                            np.array([np.log(this_S0)])),
+                            np.array([-np.log(this_S0)])),
```

Comment:
@arokem! Don´t worry, this was not a bug! I had to change the sign here because I just changed the sign of DKI's design matrix so that we can use DTI's non-linear least squares fits.